libs/blocks/caas-config/caas-config.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/exhaustive-deps */
```

Comment:
The code compat check currently fails: https://github.com/adobecom/milo/actions/runs/5881564701/job/15970376456?pr=1114#step:3:30

Looks like you need to explicitly add `react-hooks` to the `plugins` array in `.eslintrc-code-compatibility.js` (found in https://stackoverflow.com/questions/59611822/definition-for-rule-react-hooks-exhaustive-deps-was-not-found).

Reply:
👍 Thx.  I'm not actually sure we need to run this code compat step as it should be covered by the eslint pass, but will have to defer investigating for now.